radio/util/crossfire-parse.py

```diff
@@ -70,28 +71,47 @@ def dump(data, maxLen=None):
     0xAD, 0x78, 0xD2, 0x07, 0x53, 0x86, 0x2C, 0xF9
 ]
 
+# logs data are BigEndian
+def getCrossfireValue(signed, data=[], *args):
+    hexstring = "0x"
+    for n in data:
+        hexstring += hex(n)[2:]
+    value = BitArray(hex=hexstring)
+    if (signed is True):
+        return value.int
+    else:
+        return value.uint
+
 def crc8(buffer):
     crc = 0
     for c in buffer:
         crc = crc8tab[crc ^ c]
     return crc
 
+# note : python arrays slicing is a bit strange payload[x:y] means from 'x' included to 'y' NOT included
```

Comment:
why is it strange? I prefer you remove this comment ;)

Reply:
Well, it seems a lot of peop;le (including myself) bump into this all the time with python slicing, but I'll make it politicaly correct :)